### ISSUE TITLE
Solr 9.x

### ISSUE THREAD

**Issue by jamesla** [apache/solr-operator#741]
Solr 9.x has been out for a while yet this project hasn't had an update.

Is there any idea in terms of then that will be released? Is there an alternative way to deploy version 9 to production?

**Comment by janhoy** [comment]
You can deploy any 9.x version with the operator, just specify the image tag. We have kept 8.x back compat for a while, but guess next operator release can let go of that.

**Comment by gerlowskija** [comment]
Yep, there's a [Version Compatibility Matrix](https://github.com/apache/solr-operator/blob/main/docs/upgrade-notes.md#version-compatibility-matrixes) that shares this info in greater detail, but Operator 0.8.1 already supports all Solr 9.x versions (as well as the latest 8.11.x release).

Will close this out and you can re-open if there are other questions.  For next time, this would be a great question for either Solr's "user" or "dev" mailing lists.  Find info on subscribing to these [here](https://solr.apache.org/community.html#mailing-lists-chat).
